connutils_test.go

```diff
@@ -217,17 +217,18 @@ func TestConUtils(t *testing.T) {
 			dsn:  "edgedb://user@host1,host2/db",
 			expected: Result{
 				err: &configurationError{},
-				errMessage: `edgedb.ConfigurationError: ` +
-					`invalid host: "host1,host2"`,
+				errMessage: `edgedb.ConfigurationError: invalid DSN: ` +
+					`edgedb.ConfigurationError: invalid host: "host1,host2"`,
```

Comment:
I don't think we need configuration error in here twice. Maybe use `fmt.Errorf()` or something for the inner error.